File: PassPast/PassPast/Data/DataModels/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PassPast.Data.DataModels
{
	public class Comment : BaseDataModel
	{
		public Question Question { get; set; }
		public string Content { get; set; }
		public int VoteCount { get; set; }
		public DateTimeOffset Timestamp { get; set; }
		public User CreatedBy { get; set; }
		public ICollection<User> VotedBy { get; set; }
	}
}

File: PassPast 98/PassPast.Web/App/src/app/courses/models/comment.ts
import { Question } from './question';
import { Vote } from './vote';
import { User } from './user';

export interface Comment{
    id: number;
    content: string;
    questionId: number;
    createdAt: string;
    hasVoted: boolean;
    votesSum: number;
    createdBy: User;
}
